### PR TITLE
Add explanations for lenet5

### LINKED ISSUE #5649: Is the LeNet5 implementation in documentation correct?
- LeNet5 images in the paper
    ![LeNet-5 architecture as published in [1]](https://cdn-images-1.medium.com/max/2000/1*1TI1aGBZ4dybR6__DI9dzA.png)
- Current document in Chainer
    https://docs.chainer.org/en/stable/examples/cnn.html#lenet5
- What I think to be correct implementation
    http://nocotan.github.io/chainer/2017/08/04/chainercnn-copy.html

### PR DESCRIPTION
**Pull request by keisuke-umezawa**
To fix #5649 .

### CODE REVIEW

File: docs/source/examples/cnn.rst [hunk 1]
```diff
@@ -36,6 +36,9 @@ LeNet5
 ''''''
 
 Here, let's start by defining LeNet5 [LeCun98]_ in Chainer.
+In this example, we show 
+`a simplified version of LeNet5 <http://deeplearning.net/tutorial/lenet.html#lenet>`_
+that is explained in DeepLearning 0.1 documentation.
```
Comment (beam2d) [inline docs/source/examples/cnn.rst]: I'm wondering if the name "DeepLearning 0.1 documentation" is good for referring the site. "Deep Learning Tutorials" may be better.
```suggestion
introduced in Deep Learning Tutorials.
```

File: docs/source/examples/cnn.rst [hunk 2]
```diff
@@ -36,6 +36,9 @@ LeNet5
 ''''''
 
 Here, let's start by defining LeNet5 [LeCun98]_ in Chainer.
+In this example, we show 
+`a simplified version of LeNet5 <http://deeplearning.net/tutorial/lenet.html#lenet>`_
+introduced in Deep Learning Tutorials.
```
Comment (mitmul) [inline docs/source/examples/cnn.rst]:
```suggestion
introduced in the Deep Learning Tutorials.
```

**Comment by mitmul** [comment]
Jenkins, test this please

**Comment by toslunar** [comment]
Jenkins, test this please.

**Pull request merged**